osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -93,13 +93,7 @@ public virtual void Start()
             ensureSourceClockSet();
 
             if (!decoupledClock.IsRunning)
-            {
-                // Seeking the decoupled clock to its current time ensures that its source clock will be seeked to the same time
-                // This accounts for the clock source potentially taking time to enter a completely stopped state
-                Seek(FramedClock.CurrentTime);
```

Comment:
I also wonder if this was somehow related to audio devices taking time to pause? I'm not sure if that's a thing though... Because then it would be doing the seek for the only reason of seeking the track back to the current time.

Reply:
Possibly. We'll have to see how this plays out in CI and local testing. I've been testing these changes only via tests so will do some actual gameplay tonight to test any non-tested edge cases.